uint32 id

---

bool success
ObjectMsg object
